docs/self-managed/operational-guides/configure-flow-control/configure-flow-control.md

```diff
@@ -0,0 +1,311 @@
+higher than the limit). This value is given in seconds and can't be set
+null nor negative.
+
+The `limit` is the static value that we can configure to the write rate,
+therefore this can't be either null nor negative.
+
+The `throttling`, if enabled will additionally limit the write rate based on
+the exporting backlog.
+
+The exporting backlog is the quantity of records that were already
+processed but not yet exported. The increase of this is usually due to a
+mismatch of the rate of processed and exported records, or to exporting
+capability being degraded.
+
+The `throttling` algorithm used takes into account the ratio
+between the acceptable backlog and the actual backlog. If the acceptable
+backlog is at least twice as long as the real backlog we set the current
+rate as the limit (since we still have quite a margin, then we can increase
+the current processing rate to the defined limit). If the ratio between these
+is smaller than 2, then we calculate the rate by multiplying the ratio with
+the current exporting rate, with `minimumLimit` being our floor and `limit`
+being our ceiling. The intention is to increase or decrease proportionally the rate
+in between the defined limits according to the ratio of the acceptable
+backlog and the actual backlog, therefore if the acceptable backlog is
+larger than the backlog, we increase the rate, and we do the opposite if the
+backlog is larger than the acceptable backlog. From this we should expect to see the
+backlog stabilize around the acceptable backlog size (this only if the processing speed
+continues to outpace the exporting speed).
+
+The `resolution` value refers to the frequency that we adjust the
+throttling, and is given in seconds. Adjusting this value is useful to set
+the speed to which the processing rate can respond to changes.
+
+The current exporting rate is calculated by observing the number of records
+that we export during a period of time that's at least as long as the
+resolution window divided by the elapsed time.
+
+## Using the Flow Control endpoint to configure the write rate limits
+
+This feature can be especially useful adjust the flow control on the fly to
+solve an incident for a specific cluster without having to reset the clusters.
+
+:::caution
+Important to note that this should be used as a temporary fix, and changes
+should be reverted after the issue is addressed. Otherwise, permanent
+should be done through the environment variables.
+
+Another detail is that configuring the flow control through the endpoint
+does not preserve the
+configuration in the broker state, so if this restarts, any leader
+partition in this broker will revert to the defined configuration in the
+environment variables.
+:::
+
+## Fetch current configuration
+
+Note that the backup API can be reached via the /actuator management port, which by default is 8092. The configured context path does not apply to the management port.
```

Comment:
:x: The default is 9600

Reply:
Bad mistake, one of the pages I was following for structure was one from optimise, which uses this port 😅